.github/workflows/tav-command.yml

```diff
@@ -0,0 +1,103 @@
+name: tav-command
+
+on:
+  pull_request_review:
```

Comment:
[pull_request_review](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review) worked like a charm but [pull_request_review_comment](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review_comment) didn't get triggered. In any case `pull_request_review` is the one linked to a particular sha commit, hence `github.event.pull_request.head.sha` is the correct data to be used when checking out the source code.